recipes/homer/meta.yaml

```diff
@@ -11,12 +11,12 @@ package:
 
 source:
   - url: http://homer.ucsd.edu/homer/configureHomer.pl
-    sha256: 37e36fb10387a90050251683d35fe568657941b863b2088dcb0d09d9b1b59477
-    patches:
-      - configureHomer.patch
+    sha256: ccdaa3004a0e0df0882634671d4a1acc88364761e0e6c7ea329ebbf1eb729537
+    #patches:
+    #  - configureHomer.patch
```

Comment:
Please remove these lines. The .patch file is removed anyway

Reply:
Thanks, I forgot that!